deon/assets/examples_of_ethical_issues.yml

```diff
@@ -84,10 +92,8 @@
       url: https://www.liebertpub.com/doi/pdf/10.1089/big.2016.0047
     - text: Google's speech recognition software doesn't recognize women's voices as well as men's.
       url: https://www.dailydot.com/debug/google-voice-recognition-gender-bias/
-    - text: Facial recognition software is significanty worse at identifying people with darker skin.
-      url: https://www.theregister.co.uk/2018/02/13/facial_recognition_software_is_better_at_white_men_than_black_women/
-    - text: -- Related academic study.
-      url: http://proceedings.mlr.press/v81/buolamwini18a.html
+    - text: Amazon scraps AI recruiting tool that showed bias against women.
```

Comment:
This is clearly both fairness across groups and proxy discrimination, but we often use it to talk about why you can't just remove a variable from your model to get rid of its impact, so let's move back to proxy discrimination